modules/ob-authentication-to-container-registries.adoc

```diff
@@ -0,0 +1,51 @@
+// This module is included in the following assembly:
+//
+// builds/authenticating-build-images.adoc
+
+:_content-type: PROCEDURE
+[id="ob-authentication-to-container-registries_{context}"]
+= Authentication to container registries
+
+To push images to a private container registry, you must define a secret in the `OpenShift-Builds` namespace and then reference it in your `Build` custom resource (CR).
+
+.Prerequistes
+
+* 
```

Comment:
@apoorvajagtap Do we have any prerequisites for this procedure?

Reply:
There isn't a mandatory pre-requisite. One should just have the openshift build operator, relevant CRs, oc and account on registries/github, which is pretty obvious, so, I don't see any need to specify these.